packages/gimbal/src/module/lighthouse/output.ts

```diff
@@ -7,6 +7,8 @@ import { mkdirp, resolvePath, writeFile } from '@/shared/utils/fs';
 
 const type = 'lighthouse';
 
+export const checkValue = (value: number, threshold: number): boolean => value >= threshold;
```

Comment:
Maybe use the more generic threshold checking from `@/shared/utils/threshold.ts`?